### ISSUE TITLE
DetailsList(without column header) with Narrator + Edge when using Scan mode, table is not identified by narrator with table shortcut i.e. 'T'

### ISSUE THREAD

**Issue by manishsbh** [microsoft/fluentui#15020]
### Environment Information

URL: https://fabricweb.z5.web.core.windows.net/oufr/7.76.2/#/examples/detailslist/basic
Browser: Edge
OS: Windows 10

### Describe the issue:
When the DetailsList-Basic is used without the column header, it is not being identified as a table by the Narrator in the Edge browser. It is also not responding to the table shortcut 'T'. Same is the case with DetailsList-Large Grouped, even there the table is not being identified. But the narrator announces the table details when column header is made visible.

### Please provide a reproduction of the issue in a codepen:
Reproduction URL: https://codepen.io/manishsbhoopalam/pen/WNwyzrG?editors=0010

Reproduction Steps:
1. Open the above link
2. Start Narrator
3. Navigate to the Text Input Box (Filter by Name)
4. Press 'T' (shortcut for identifying the next table)

#### Actual behavior:

Narrator announces "No next table"

#### Expected behavior:

Narrator must announce the table details and navigate to the first row of the list

**Comment by paulgildea** [comment]
@manishsbh Thanks for the feedback.

@smhigley - Any insight here on the right behavior?

**Comment by kolaps33** [comment]
I was investigating it a bit today and observed following:

I tested it in scan mode and I hadn't focus in  input/text field. Because if focus is in the  input/text field then press "T" will type "T" key in the input field. So I was rather in scan mode on the text upper to input/text field. And Table navigation with T works only when scan mode is on. 

I tried the same example on the Chrome and it was working with Narrator as expected. It navigates to the table. 
As well it works with JAWS and NVDA in the Chrome. 

As well Narrator/Edge navigates to the table(Grid) on the page:
https://fabricweb.z5.web.core.windows.net/oufr/7.76.2/#/examples/detailslist/basic

When I tried to put simple <table> into codepen, then Narrator navigates into table.

So from my observation there is issue somewhere between fabric code+codepen+Edge+Narrator, but I was not able to easy identify where could be problem. 

I would take this bug seriously if is reproducible on some other page, not only in the codepen. If some consumer of fabric(FluentUI) using it?
